bin/main.rb

```diff
@@ -0,0 +1,5 @@
+# A file which has a shebang line as its first line is
+# granted execute permission.
+
+# !/usr/bin/env ruby
+puts 'hello, world'
```

Comment:
- [ ] Now, for the permissions for the file, if both of you have windows, you can not use the chmod command. So let's make use of a "virtual environment" with linuxbrew (this is called a pod in DevOps, you can research about it later if you want). Basically, it gives you a blank computer in the cloud with ruby and Linux previously installed, just append `gitpod.io/` to your repo URL, in this case: https://gitpod.io/github.com/edxco/tic-tac-toe/tree/project_setup

  - Log in with your github account
  - In a new tab, go to https://gitpod.io/access-control/ and give permission to write public repos
  - go back to the tab with your project in gitpod and make changes as you would in your computer, follow the instructions in the [article of pathwright](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/)
  - git add, git commit and git push :)